pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -339,10 +364,10 @@ func (lb *ListenerBuilder) getListeners() []*xdsapi.Listener {
 			listeners = append(listeners, lb.virtualInboundListener)
 		}
 
-		log.Debugf("Build %d listeners for node %s including %d inbound, %d outbound, %d virtual and %d virtual inbound listeners",
+		log.Debugf("Build %d listeners for node %s including %d outbound, %d virtual and %d virtual inbound listeners",
```

Comment:
Is the first "virtual" in `"%d virtual and %d virtual inbound listeners"` referring to the virtual outbound listeners? could you update the log to make it clear.